lib/exercises.rb

```diff
@@ -1,29 +1,85 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
-
+# Time Complexity: O(n) where n is the number of chars
+# Space Complexity: O(n) where n is the number of words
 def grouped_anagrams(strings)
```

Comment:
👍 Interesting solution, the time complexity is O(n) if the words are pretty short, if they're unlimited in length the time complexity is O(nm) where m is the length of the longest string.